src/utils/ProjectUtils.hx

```diff
@@ -189,18 +189,25 @@ class ProjectUtils
 		}
 		return false;
 	}
-	
+
 	public static function openWithIntelliJIDEA(projectPath:String, projectName:String):Bool
 	{
 		var projectFile = CommandUtils.combine(projectPath, ".idea");
-		
+
 		if (FileSys.exists(projectFile))
 		{
-			if (FileSys.isMac || FileSys.isLinux) // TODO: test on linux
+			if (FileSys.isMac)
+			{
+				Sys.command("open", ["-a",FlxTools.settings.IDEA_Path, projectPath]);
+			}
+			else	if (FileSys.isLinux || FileSys.isWindows) // TODO: test on linux
```

Comment:
Why is this `|| FileSys.isWindows`? I don't think that will work, with `disown`? Besides, doesn't that make the following if useless?